tests/clients/web/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 # All of our core javascript will be checked with a single test
-add_javascript_style_test(core src)
```

Comment:
If you do start linting the test files, you can solve part of https://github.com/girder/girder/issues/944.